src/pages/start/q/integration/[integration].mdx

```diff
@@ -10,8 +10,7 @@ The open-source Amplify provides the following products to build fullstack iOS,
 - **Amplify [CLI](/cli)** - Configure all the services needed to power your backend through a simple command line interface.
 - **Amplify [Libraries](/lib)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
 - **Amplify [UI Components](/ui)** - UI libraries for React, React Native, Angular, Vue and Flutter.
-
-The **Amplify [Hosting](https://aws.amazon.com/amplify/hosting/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
+- The **Amplify [Hosting](https://aws.amazon.com/amplify/hosting/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
```

Comment:
Should "The" be removed before the Amplify Hosting link? 

It reads weird where "The Amplify Hosting is an AWS service..."

I think one other way to say it could be "The Amplify Hosting Console is an AWS service...", but up to you!